src/App.tsx

```diff
@@ -1,51 +1,81 @@
+import classNames from 'classnames';
 import React from 'react';
 import './App.scss';
 
-// import goodsFromServer from './goods';
-
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
-
-        <button
-          type="button"
-          className="App__clear"
-        >
-          Clear
-        </button>
-      </header>
-
-      <ul>
-        <li className="Good">Dumplings</li>
-        <li className="Good">Carrot</li>
-        <li className="Good">Eggs</li>
-        <li className="Good">Ice cream</li>
-        <li className="Good">Apple</li>
-        <li className="Good">Bread</li>
-        <li className="Good">Fish</li>
-        <li className="Good">Honey</li>
-        <li className="Good Good--active">Jam</li>
-        <li className="Good">Garlic</li>
-      </ul>
-
-      {/* Put required buttons into each Good */}
-      <button
-        type="button"
-        className="Good__remove"
-      >
-        Remove
-      </button>
-
-      <button
-        type="button"
-        className="Good__select"
-      >
-        Select
-      </button>
-    </main>
-  );
+import goodsFromServer from './goods';
+
+type State = {
+  selectedGood: string;
 };
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
+
+  selectGood = (good: string) => {
+    this.setState({
+      selectedGood: good,
+    });
+  };
+
+  removeHandler = () => {
+    this.setState({
+      selectedGood: '',
+    });
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App has-text-centered">
+        { selectedGood !== ''
+          ? (
+            <>
+              <h1 className="title is-2">
+                {selectedGood}
+                {' '}
+                is selected
+                <button
+                  type="button"
+                  className="button is-danger is-light is-outlined"
+                  onClick={this.removeHandler}
+                >
+                  Clear
+                </button>
+              </h1>
+
+            </>
+          )
+          : (<h1 className="title is-2">No goods selected</h1>)}
+
+        <ul className="content is-medium content">
+          {goodsFromServer.map((good) => (
+            <li
+              key={good}
+              className={classNames({ active: selectedGood === good })}
+            >
+              {good}
+              <button
+                type="button"
+                className={selectedGood === good
+                  ? 'button is-danger is-outlined'
+                  : 'button is-success is-outlined'}
```

Comment:
use https://www.npmjs.com/package/classnames this lib in next tasks for this purpose